### PR TITLE
Setup endpoints for EB and optimized existing code

### PR DESCRIPTION
**Pull request by SVijayB**
### Description

Setup endpoints for adding, updating, deleting, and querying EB member details.

### Motivation and Context

The current version of displaying EB member details in https://www.oscvitap.org/team is not dynamic. 
Adding EB member details to the API helps in rendering the page easier and also adds scalability to the current website.
The API can also be used in OSC-DBot (https://github.com/SVijayB/OSC-DBot/issues/14) for getting EB details.
Apart from sending POST requests, I've set up a basic file upload form page for direct and ease of access.

This does require an update to the docs which will be worked on soon. Referencing #7. 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Review by kode-logger — APPROVED** [review]
Everything seems good, if this patch has cleared all the test trials then we can go ahead and merge it.

File: scripts/csv_to_json.py [hunk 1]
```diff
@@ -45,19 +47,31 @@ def csv_to_json(csvFile, jsonFile):
...
+                        position_data[key].append(rows)
                     else:
-                        json_data[key] = [rows]
+                        position_data[key] = [rows]
+
+    for key in json_data.keys():
+        for i in range(len(json_data[key])):
+            del json_data[key][i]["Position"]
+
+    dept_list = heirarchy_order.keys()
```
Comment (dat-adi) [inline scripts/csv_to_json.py:67]: Might wish to correct the spelling of the word,
`heirarchy` => `hierarchy`.

**Review by dat-adi — APPROVED** [review]
Other than a grammatical correction, everything else looks good to go, changes are neat, and are approved to go.

File: scripts/csv_to_json.py (hunk 1 above)
Comment (SVijayB) [reply scripts/csv_to_json.py]: I did make that mistake in a lot of places xD. Made changes as requested.

**Pull request merged**